wagtail/admin/tests/test_privacy.py

```diff
@@ -211,6 +211,19 @@ def test_unset_password_restriction(self):
             expected_log_message,
         )
 
+    @override_settings(WAGTAIL_ALLOW_SHARED_PASSWORD_PAGE=False)
+    def test_disable_shared_password_page(self):
+        response = self.client.get(
+            reverse("wagtailadmin_pages:set_privacy", args=(self.public_page.id,)),
+        )
+        self.assertEqual(response.status_code, 200)
+        self.assertNotIn("password", response.context["form"].fields)
+        self.assertFalse(
```

Comment:
Suggestion: It'd be nice to have a test that neither the password input nor the option for password are visible